developer/src/kmc-keyboard/package.json

```diff
@@ -24,9 +24,9 @@
   "dependencies": {
     "@keymanapp/keyman-version": "*",
     "ajv": "^8.11.0",
-    "restructure": "^3.0.0",
+    "restructure": "git+https://github.com/keymanapp/dependency-restructure.git#49d129cf0916d082a7278bb09296fb89cecfcc50",
```

Comment:
.. why?

Reply:
no PR in dependency-restructure's upstream, and xml2js says they would accept the PR if it had tests.. might be worth while

Reply:
Maybe I'd say it this way.  I'd prefer if we had to do this that it was a commit that was a PR into https://github.com/foliojs/restructure - it's close!